vsintegration/src/FSharp.Editor/Hints/InlineParameterNameHints.fs

```diff
@@ -30,14 +30,19 @@ module InlineParameterNameHints =
     let private doesFieldNameExist (field: FSharpField) = 
         not field.IsNameGenerated
 
-    let isMemberOrFunctionOrValueValidForHint (symbol: FSharpMemberOrFunctionOrValue) =
-        // is there a better way?
-        let isNotBuiltInOperator = 
-            symbol.DeclaringEntity 
-            |> Option.exists (fun entity -> entity.CompiledName <> "Operators")
-
-        symbol.IsFunction
-        && isNotBuiltInOperator // arguably, hints for those would be rather useless
+    let isMemberOrFunctionOrValueValidForHint (symbol: FSharpMemberOrFunctionOrValue) (symbolUse: FSharpSymbolUse) =
+        // make sure we're looking at a call site and not the definition
+        if symbolUse.IsFromUse then
+            // is there a better way?
+            let isNotBuiltInOperator = 
+                symbol.DeclaringEntity 
+                |> Option.exists (fun entity -> entity.CompiledName <> "Operators")
+
+            (symbol.IsFunction && isNotBuiltInOperator) // arguably, hints for those would be rather useless
```

Comment:
At this place, the comment becomes confusing a bit. Maybe just remove it, this was just a point for a discussion in the initial PR :)

Reply:
We can adjust comments in followups, resolving this for now